pytorch/inference/docker/1.12/py3/cu116/Dockerfile.gpu

```diff
@@ -282,6 +282,8 @@ ARG PYTHON
 
 ENV SAGEMAKER_SERVING_MODULE sagemaker_pytorch_serving_container.serving:main
 
+RUN pip install -U pyOpenSSL cryptography
```

Comment:
are these needed for the tests? what is the reason to update these dockerfiles? If it's not related to the tests, can you just update the description of the PR?